eyes2-lib/src/entity/genotype/genotype.rs

```diff
@@ -13,7 +11,7 @@ pub enum BadGenomeError {
 // expected that the Genotype will be defined by a genome, and that the
 // genome (with mutations as appropriate) will be passed to the
 // descendant creatures.
-pub trait Genotype {
+pub trait Genotype: erased_serde::Serialize {
```

Comment:
This trait bound shouldn't be required afaik

Reply:
apparently `erased_serde::serialize_trait_object!(Genotype)` requires it.